back-end/YummyGen/YummyGenAPI/Startup.cs

```diff
@@ -17,13 +19,25 @@ public static void RegisterServices(WebApplicationBuilder builder)
             var connectionString = builder.Configuration.GetConnectionString("WebApiDatabase");
             builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseNpgsql(connectionString));
 
+            builder.Services.AddIdentity<User, IdentityRole>(options =>
```

Comment:
Good job. I never knew that you can use such a powerful tool for passwords. I will consider now using this in feature projects. Ty